docs/source/io_formats/tallies.rst

```diff
@@ -154,14 +154,30 @@ For each filter type, the following table describes what the ``bins`` attribute
 should be set to:
 
 :cell:
-  A list of unique IDs for cells in which the tally should be accumulated.
+  A list of cells or unique IDs for cells in which the tally should be 
```

Comment:
My bad. Still not used to not seeing the Python API as the only way to input parameters